api/src/main/java/org/openmrs/ui/framework/FormatterImpl.java

```diff
@@ -115,6 +117,13 @@ private boolean wholeNumber(Number n) {
 	
 	private String format(Date d, Locale locale) {
 		DateFormat df;
+		boolean convertTimezones = BooleanUtils
+		        .toBoolean(administrationService.getGlobalProperty(UiFrameworkConstants.GP_TIMEZONE_CONVERSIONS));
```

Comment:
I see `clientTimezone` used in more than one place. Do we wanna define a constant for it?

Reply:
It's used twice, if you think its worth, sure i can do it.

Reply:
Created a constant for it.